valhalla/sif/edgelabel.h

```diff
@@ -519,7 +534,8 @@ class PathEdgeLabel : public EdgeLabel {
                 const InternalTurn internal_turn,
                 const uint8_t path_id = 0,
                 const bool destonly = false,
-                const bool hgv_access = false)
+                const bool hgv_access = false,
+                const uint8_t destonly_access_restr_mask = 0)
```

Comment:
do we need this on _all_ classes? I guess at least this one not, only used for recosting